### DISCUSSION TITLE
3.44.0

### DISCUSSION THREAD

**Discussion by mifi** [mifi/lossless-cut#1065] [General]
🎉 Most requested feature: 🎊

**Implement smart cut (experimental) 🔥 #126**

Other improvements and fixes:
- build for linux arm64 (without ffmpeg) #1056
- improve format detection in concat/merge dialog and allow selecting any output format #1032
- auto-convert any subtitle to mov_text when output is mp4 #418
- auto convert subtitle mov_text to srt when trying to put it in matroska (and webm to webvtt)
- default to mov when pcm audio - fixes #947 #163 #367 #948
- losslessly convert pcm_bluray to pcm_s24le for non-mpegts output formats - fixes #476, #895
- increase size of concat dialog
- re-introduce merge menu item
- for mp4/mov, use vtag hvc1 instead of the default unsupported hev1 #1032
- copy disposition when concat (ffmpeg doesnt automatically)
- fix color bug
- remove menu from batch list (less clicks)
- improve batch list selection
- move sorting from merge dialog to batch files list #1043
- make sure to select previous item when deleting from batch list #89 (comment)
- improve canvas player to hopefully workaround losslesscut keeping file handles #272
- improve concat ffmpeg command logging #954
- add key binding for fix invalid duration #254
- allow using external ffmpeg executable #400
- fix play button #1048
- implement increase/descrease volume hotkeys #254
- try to fix issue where file does not open on macos (Open With LosslessCut)
- implement function to shift all segments times
- allow configuring where to store project file #1004
- chapters improvments #993 (with chapters only export, add gap chapters between segments)
- segment list allow selecting multiple
- implement "select segments by label" #993
- implement function to fill gaps between segments #993
- implement function to remove multiple segments
- Implement Extract all frames as images #1033
- implement label selected segments #993
- move merge options into separate dialog and add option to clear batch
- fix missing keyframe seek shortcuts  #1038

<hr /><em>This discussion was created from the release <a href='https://github.com/mifi/lossless-cut/releases/tag/v3.44.0'>3.44.0</a>.</em>

**Comment by oliko-tk** [comment]
GRACIAS POR TU ESFUERZO Y TRABAJO.M ES MUY BUENO  EL PROGRAMA 😁😁

**Comment by www-MKRD-info** [comment]
AWESOME!!!

**Comment by mbs** [comment]
I'm using 3.39.0 presently; tried upgrading to 3.44.0 but the preview performance (using keyboard to seek) is much poorer, so I suppose I will stick with 3.39 for now

Edit: I stepped back through more recent versions and it seems that performance issue was introduced between 3.42 and 3.43, so now I'll begin using 3.42.

**Reply by mifi** [reply]
thanks for researching. that's valuable knowledge, I'm adding it here: #1097
